content/webapp/services/prismic/transformers/body.ts

```diff
@@ -436,6 +416,8 @@ function transformContentListSlice(slice: ContentListSlice): BodySlice {
               return transformSeason(content);
             case 'card':
               return transformCard(content);
+            default:
+              return undefined;
```

Comment:
linting fix: `Array.prototype.map() expects a value to be returned at the end of arrow function`